### PR TITLE
Updated reality-data-client to use iTwinjs v4

### PR DESCRIPTION
**Pull request by mcote01**
also updated version to 1.0.0-dev.3
for dev dependencies use  `npm i --legacy-peer-deps`

### CODE REVIEW

**Comment by MarcBedard8** [comment]
@aruniverse should that be "@itwin/core-geometry": "^4.1.0-dev.0", or "@itwin/core-geometry": "^4.0.0"? In other word, should we target 4.0 or 4.1 for our 1.0 release of reality-data-client?

**Comment by aruniverse** [comment]
> @aruniverse should that be "@itwin/core-geometry": "^4.1.0-dev.0", or "@itwin/core-geometry": "^4.0.0"? In other word, should we target 4.0 or 4.1 for our 1.0 release of reality-data-client?

i would recommend 4.0
since it hasnt released you can use `^4.0.0-dev.105` (the latest rc)

**Comment by mcote01** [comment]
> > @aruniverse should that be "@itwin/core-geometry": "^4.1.0-dev.0", or "@itwin/core-geometry": "^4.0.0"? In other word, should we target 4.0 or 4.1 for our 1.0 release of reality-data-client?
> 
> i would recommend 4.0 since it hasnt released you can use `^4.0.0-dev.105` (the latest rc)

like this?

File: package.json [hunk 1]
```diff
@@ -42,25 +42,25 @@
     "url": "http://www.bentley.com"
   },
   "dependencies": {
-    "@itwin/core-geometry": "^3.7.0",
+    "@itwin/core-geometry": "^4.0.0-dev.105",
     "axios": "^1.3.6",
     "util": "^0.12.5"
```
Comment (aruniverse) [inline package.json:47]: i dont think we need this dep on util at all

**Review by MarcBedard8 — APPROVED** [review]

**Pull request merged**